src/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+# ===========================================
+# Set up navier-stokes library
+
+add_library(navier_stokes INTERFACE "navier_stokes/navier_stokes.h")
+add_library(feltor::navier_stokes ALIAS navier_stokes)
+target_link_libraries(navier_stokes INTERFACE dg::dg)
```

Comment:
You can leave everything from the navier_stokes project out of the feltor project. It is not a library. Rather, it is its own executable that happens to use the same main function as feltor and I think should have its own CMakeLists.txt file. I think i have enough knowledge now that I can set it up myself, once the pull-request is merged